combat-trainer.lic

```diff
@@ -3530,13 +3544,17 @@ class GameState
   end
 
   def wield_whirlwind_offhand
+    return if weapon_skill.include?'Twohanded'
     return unless currently_whirlwinding
     @equipment_manager.wield_weapon_offhand(whirlwind_offhand_name)
   end
 
   def determine_whirlwind_action
-    if @use_weak_attacks && Flags['ct-accuracy-ready']
-      update_analyze_array(true) if @analyze_combo_array.empty?
+    if @flame_for_expertise == true && ( (DRStats.mana < @flame_inner_fire_threshold) || (DRSkill.getxp('Expertise') < 15) )
```

Comment:
Since `@flame_for_expertise` is a boolean, we can remove the ` == true` part of the expression for brevity, right?

```suggestion
    if @flame_for_expertise && ( (DRStats.mana < @flame_inner_fire_threshold) || (DRSkill.getxp('Expertise') < 15) )
```